### PR TITLE
Check Lego: Filter AWS EBS Volume with Low IOPS's

### PR DESCRIPTION
**Pull request by harshal-netserv**
## Description
Filter AWS EBS Volume with Low IOPS's

### Testing
image

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from unskript.legos.aws.aws_list_all_regions.aws_list_all_regions import aws_list_all_regions
+import pprint
+
+
+class InputSchema(BaseModel):
+    region: Optional[str] = Field(
+        default="",
+        title='Region',
+        description='AWS Region.')
+    iops_threshold: Optional[int] = Field(
```
Comment (dougsillars) [inline AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py:16]: si region and iops really optional?

If they are not included will the Lego run?

If you convert Optional[str] to str - they become required.

File: AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                if volume.iops < iops_threshold:
+                    volume_dict["region"] = reg
+                    volume_dict["volume_id"] = volume.id
+                    volume_dict["volume_iops"] = volume.iops
+                    result.append(volume_dict)
+        except Exception as e:
+            pass
+
+    if len(result) != 0:
+        return (False, result)
```
Comment (dougsillars) [inline AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py:63]: I dont understand why the true/false.  I think just returning the list is ok.. If it is empty - its clear there are none.. If there is one - then we need to do something,

File: AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py (hunk 1 above)
Comment (dougsillars) [reply AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py:16]: Maybe in the readme say - if not provided - the value is set to 100

**Review by dougsillars — CHANGES_REQUESTED** [review]
it looks good. I think I now understand why the inputs are optional - bt maybe say something like - if not provided "x" will be used.

Also - the response can jsut be the list. the boolean doesnt make sense to me.

File: AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py (hunk 2 above)
Comment (harshal-netserv) [reply AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py:63]: I created this Lego as check Legos as per the discussion the output for check Lego should be in tuple only.

File: AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py (hunk 1 above)
Comment (harshal-netserv) [reply AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py:16]: I will add this point in the description and yes if both inputs are not provided then also Lego will run.

**Review by dougsillars — APPROVED** [review]

**Pull request merged**